src/controllers/routes/http.routes.ts

```diff
@@ -1,27 +1,27 @@
 import { Router } from '@well-known-components/http-server'
 import { GlobalContext } from '../../types'
 import { errorHandler } from '@dcl/platform-server-commons'
-import { getCommunityHandler } from '../handlers/get-community-handler'
-import { getCommunitiesHandler } from '../handlers/get-communities-handler'
-import { deleteCommunityHandler } from '../handlers/delete-community-handler'
+import { getCommunityHandler } from '../handlers/http/get-community-handler'
+import { getCommunitiesHandler } from '../handlers/http/get-communities-handler'
+import { deleteCommunityHandler } from '../handlers/http/delete-community-handler'
 import { wellKnownComponents } from '@dcl/platform-crypto-middleware'
-import { getCommunityMembersHandler } from '../handlers/get-community-members-handlers'
-import { getMemberCommunitiesHandler } from '../handlers/get-member-communities-handlers'
-import { removeMemberFromCommunityHandler } from '../handlers/remove-member-from-community-handler'
-import { createCommunityHandler } from '../handlers/create-community-handler'
-import { addMemberToCommunityHandler } from '../handlers/add-member-to-community-handler'
-import { unbanMemberHandler } from '../handlers/unban-member-handler'
-import { banMemberHandler } from '../handlers/ban-member-handler'
-import { getBannedMembersHandler } from '../handlers/get-banned-members-handler'
-import { createReferralHandler } from '../handlers/create-referral-handler'
-import { updateReferralSignedUpHandler } from '../handlers/update-referral-signed-up-handler'
-import { getInvitedUsersAcceptedHandler } from '../handlers/get-invited-users-accepted-handler'
-import { updateMemberRoleHandler } from '../handlers/update-member-role-handler'
+import { getCommunityMembersHandler } from '../handlers/http/get-community-members-handlers'
+import { getMemberCommunitiesHandler } from '../handlers/http/get-member-communities-handlers'
+import { removeMemberFromCommunityHandler } from '../handlers/http/remove-member-from-community-handler'
+import { createCommunityHandler } from '../handlers/http/create-community-handler'
+import { addMemberToCommunityHandler } from '../handlers/http/add-member-to-community-handler'
```

Comment:
What if we use the index in the HTTP handlers route and impor them all from there?